eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -1,5 +1,9 @@
 #Requires -Version 7.0
 $CI_YAML_FILE = "ci.yml"
+
+#mgmt: resourceProvider to sdk package name map
+$packageNameHash = [ordered]@{"vmware" = "avs"; "azure-kusto"="kusto"; "cosmos-db"="cosmosdb";"customer-insights"="customerinsights";"monitor"="insights";"msi"="managedserviceidentity";"web"="appservice"}
```

Comment:
It seems like resourceProvider to sdk service directory mapping? If so, the service directory for appservice is `websites`, not `appservice`.

Reply:
@fengzhou-msft would you please provide the full mappings? I just got it from previous script.